index.html

```diff
@@ -2,6 +2,8 @@
 ---
 {% include header.html %}
 
+<!--Google Translate Plugin-->
```

Comment:
Why this comment?

Reply:
small mistake. This plugin was this place earlier before done requested